<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDiagnosticTab</class>
 <widget class="QWidget" name="SettingsDiagnosticTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="settings_diagnostic_tab_layout">
     <item>
      <widget class="QLabel" name="select_row_label">
       <property name="text">
        <string>Select row: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="select_row_combo_box"/>
     </item>
     <item>
      <widget class="QPushButton" name="select_row_push_button">
       <property name="text">
        <string>Update </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="expand_button">
       <property name="text">
        <string>Expand</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="collapse_button">
       <property name="text">
        <string>Collapse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tree_widget">
     <property name="animated">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>350</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>350</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">Key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="save_state_group_box">
     <property name="title">
      <string>Save current state to file</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="save_state_label">
        <property name="text">
         <string>Save to</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="save_state_browse_push_button">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="save_state_line_edit"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="save_state_save_push_button">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
